drracket/scribblings/drracket/menus.scrbl

```diff
@@ -543,3 +543,47 @@ brings the corresponding window to the front.
    DrRacket to other languages.}
 
 ]
+
+
+@; ----------------------------------------
+
+@section{@onscreen{Context menu}}
+
+The following additional menu items may be available when activating the context menu in an 
+editing panel. 
+
+@itemize[
+
```

Comment:
Some of these items appear in some context and some in others. How about breaking them up into lists based on a description of where they might appear?

Reply:
Go for it, but I would not worry too much